studymanager-core/src/main/java/io/redlink/more/studymanager/core/ui/ViewConfig.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright LBI-DHP and/or licensed to LBI-DHP under one or more
+ * contributor license agreements (LBI-DHP: Ludwig Boltzmann Institute
+ * for Digital Health and Prevention -- A research institute of the
+ * Ludwig Boltzmann Gesellschaft, Österreichische Vereinigung zur
+ * Förderung der wissenschaftlichen Forschung).
+ * Licensed under the Elastic License 2.0.
+ */
+package io.redlink.more.studymanager.core.ui;
+
+import java.util.List;
+
+/**
+ * Represents the configuration of a data view.
+ *
+ * @param filters the list of filters applied to the data view
+ * @param rowAggregation the aggregation method for rows
+ * @param seriesAggregation the aggregation method for series
```

Comment:
```suggestion
 * @param rowAggregation the aggregation method for rows. In most cases, this will be the data-legend in the view/chart.
 * @param seriesAggregation the aggregation method for series. In most cases, this will result in the x-axis of the view/chart
```